src/styles/navstyles.css

```diff
@@ -66,10 +104,15 @@
     background: var(--color-navyblue);
   }
   
-  .navbar-link {
-    text-decoration: none;
+  .navbar-link, .pages {
```

Comment:
Why the two names? Let's be consistent and just use the className "navbar-link".